### ISSUE TITLE
Refactoring

### ISSUE THREAD

**Issue by filiagees** [KekkaiSensen/Tetris#2]
@KekkaiSensen 
eu dei uma breve mudada no código. Vou deixar essa issue aqui para discutir coisas relacionada a mudanças no código beleza?

desafio :joy: :
- por que eu coloquei um `#-*- coding: utf-8 -*-` no começo do código do tetris.py ?
- e também, porque eu coloquei todo o código dentro de uma `def main():` ?

dica: testa rodar normal e rodar apagando essa linha

**Comment by mar-moraes** [comment]
@filiagees Em questão do #-*- coding: utf-8 -*-  rodei com e sem ele e não aconteceu nada, mas eu dei um  Google e fiquei sabendo que daria um erro por conta da codificação do código 😂 . 
Agora em relação ao main( ), até onde eu sabia é pelo fato que é parte principal do código, é a onde vai rodar o código e resto seria só componentes para ele (eu acho, não pesquisei no Google 😂😂), talvez você tenha coloca em main( ) para ter um loop infinito de peças diferentes caindo🤔...

**Comment by filiagees** [comment]
sim, se você rodasse com python2 e sem isso, seria um erro parecido com:
```
python tetris.py
  File "tetris.py", line 80
SyntaxError: Non-ASCII character '\xc3' in file tetris.py on line 80, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
```
por causa do **ã** em `não`, mesmo escrito num comentário, interessante reparar nisso.

Mas, como usei tipagem ` -> `  (python3) não iria dar para rodar no python2 de qualquer forma :joy:
